ENIGMAsystem/SHELL/Platforms/xlib/Makefile

```diff
@@ -1,5 +1,5 @@
 SOURCES += $(wildcard Platforms/xlib/*.cpp) $(wildcard Platforms/General/POSIX/*.cpp)
-override CXXFLAGS +=  -I/usr/X11R6/include -I/usr/include -I/usr/include/libpng
-override CFLAGS += -I/usr/X11R6/include -I/usr/include -I/usr/include/libpng
+override CXXFLAGS +=  -I/usr/X11R6/include -I/usr/include
+override CFLAGS += -I/usr/X11R6/include -I/usr/include
```

Comment:
cant u use pkgcong here?

Reply:
Do you mean I just do:
`override CXXFLAGS += $(pkg-config x11 --cflags --libs)
override CFLAGS += $(pkg-config x11 --cflags --libs)`

Just wanted to make sure I wrote this correctly before I commit.

Reply:
That works. Will do.

Reply:
@fundies ok. Change made and it's passing. Let me know if you run into any other objections and I'll be happy to fix it accordingly.